doc/UsersGuide/source/BuildingRunningTesting/ExtQuickStart.rst

```diff
@@ -91,7 +91,7 @@ HAFS Physics Configuration
 
 Look in ``HAFS/parm/hafs.conf`` to determine what physics suites are running.
 
-.. figure:: https://github.com/hafs-community/HAFS/wiki/docs_images/hafs_ccpp_suites.png
+.. figure:: https://raw.githubusercontent.com/wiki/hafs-community/HAFS/docs_images/hafs_ccpp_suites.png
```

Comment:
Same here. Keep links as is. The redirect in `conf.py` should take care of this.